README.rst

```diff
@@ -61,17 +61,17 @@ from adafruit_pybadger import PyBadger
 pybadger = PyBadger()
 
 while True:
-    pybadger.show_badge(hello_scale=2, my_name_is_scale=2, name_scale=3)
+    pybadger.show_badge(name_string="Blinka", hello_scale=2, my_name_is_scale=2, name_scale=3)
```

Comment:
I would recommend moving show_badge and auto_dim out of the loop and then allowing something like pressing the start button to switch back to badge mode.